0011-command-execution-protocol/README.md

```diff
@@ -0,0 +1,170 @@
+---
+name: Command Execution Protocol
+shortname: 11/CEP
+status: raw
+editor: Alexey Shmalko <ashmalko@kaaiot.io>
+---
+
+<!-- toc -->
+
+# Introduction
+Command Execution Protocol is an endpoint-aware 1/KP extension. It is designed to allow command execution on endpoints.
```

Comment:
@rasendubi In the Cons section, you listed some specifics of the contribution process related to documentation, but not the actual cons of the sentence-per-line method. These specifics remain true regardless of whether you use that method. Except for the last bullet -- I don't think it's a strong argument since we're talking about markdown.

If you ask me, I like the sentence-per-line method as it is friendlier to human eyes -- this is the main reason why we use Markdown in the first place. Now let's look at the other side -- what are the advantages of using multiple sentences per line?